libraries/bot-azure/src/test/java/com/microsoft/bot/azure/AzureQueueTests.java

```diff
@@ -129,7 +140,7 @@ private void assertEmulator() {
 
     private class ContinueConversationLater extends Dialog {
         @JsonProperty("disabled")
-        private Boolean disabled;
+        private Boolean disabled = false;
```

Comment:
Are we sure about this?

Reply:
It is required because otherwise the variable is null and fails when trying to use it.

Reply:
Ack